### ISSUE TITLE
[BUG] Cant install using pip.

### ISSUE THREAD

**Issue by adamboutcher** [graphite-project/graphite-web#2815]
**Describe the bug**
Doesn't install using pip correctly

**To Reproduce**

/usr/bin/python3 -m venv /opt/graphite
source /opt/graphite/bin/activate
pip install graphite-web
but no such file or directory: /opt/graphite/webapp/graphite

**Expected behavior**
Expected the complete tar to be there.

**Environment (please complete the following information):**
 - OS flavor: Rocky Linux 9.2
 - Graphite-web version: 1.1.10
 - Django/Python version: Python 3.9
 - Setup type: pip in a virtual env

**Additional context**
The tar file from Pypi is correct but however pip deals with this results in that directory not being there.

**Comment by deniszh** [comment]
Yes, latest pip/virtualenv are broken. For python 3.9 you can use pip==20.1.1 and virtualenv==16.7.10

**Comment by earthgecko** [comment]
Just FYI for anyone else that bumps into this.  Another method to circumvent this pip and virtualenv issue is to install this in your virtualenv as follows with no-binary

```
cd /opt/graphite
source bin/activate
bin/pip3 install --no-binary=:all: https://github.com/graphite-project/graphite-web/tarball/1.1.10
```

This works on Python 3.8, 3.9 and 3.10
